cmd/cerbos/compile/compile.go

```diff
@@ -60,26 +43,28 @@ cerbos compile --skip-tests /path/to/policy/repo
 `
 
 type Cmd struct {
-	Dir           string `help:"Policy directory" arg:"" required:"" type:"existingdir"`
-	Format        string `help:"Output format (${enum})" default:"pretty" enum:"pretty,plain,json" short:"f"`
-	Tests         string `help:"Path to the directory containing tests. Defaults to policy directory." type:"existingdir"`
-	RunRegex      string `help:"Run only tests that match this regex" name:"run"`
-	SkipTests     bool   `help:"Skip tests"`
-	IgnoreSchemas bool   `help:"Ignore schemas during compilation"`
-	Verbose       bool   `help:"Verbose output on test failure"`
+	Dir           string               `help:"Policy directory" arg:"" required:"" type:"existingdir"`
+	Output        flagset.OutputFormat `help:"Output format (${enum})" default:"tree" enum:"tree,pretty,json" short:"o"`
```

Comment:
I think `pretty` output doesn't make sense anymore. It should probably be called `list`